src/components/FaqAccordian/index.tsx

```diff
@@ -0,0 +1,92 @@
+import React from "react"
+import { MdChevronRight } from "react-icons/md"
+import * as AccordionPrimitive from "@radix-ui/react-accordion"
+
+import { cn } from "@/lib/utils/cn"
+
+import * as RootAccordion from "../../../tailwind/ui/accordion"
+
+const AccordionTrigger = React.forwardRef<
```

Comment:
We should avoid using the same names as inside the `ui/accordion` component to avoid ambiguity/confusion. Made a note above about potentially renaming the file, but in either case we should include "Faq" or "FAQ" in these component names to make it clear.